### DISCUSSION TITLE
OpenHack Uninstall

### DISCUSSION THREAD

**Discussion by POPCATLOLOLOLOLOL** [community/community#110667] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I installed OpenHack v2.01 for Geometry Dash.  Worked great, however, when I tried to install Geode for Geometry Dash it is complaining that another mod loader is installed.  Has anyone come across this?  How would I uninstall the OpenHack mod?

![moderror](https://github.com/community/community/assets/161174682/cfd1b465-f191-4fc9-b149-e6a53303677f)

Any assistance would be appreciated!

Thanks.

**Comment by 0xSecureByte** [comment]
Goto C: -> Program Files (x86) and then try to find Geode or similar folder and delete it

If you cant find it, I'm happy to help using Anydesk 

Regards.

**Comment by POPCATLOLOLOLOLOL** [comment]
Hi, thanks for your reply.  I don't see those folders.  We tried removing the OpenHack folder from the \Program Files (x86)\Steam\steamapps\common\Geometry Dash but it had no effect.  Any ideas?

**Comment by 0xSecureByte** [comment]
Just click on Browse and change the path, make it outside of "Program FIles (x86)"

It should fix it

**Comment by rverrelli** [comment]
If I choose any other path, I get:

[image: image.png]
